bin/validate-commit-msg.js

```diff
@@ -15,12 +15,11 @@ process.stdin.on('readable', () => {
   }
 });
 
-
-process.stdin.on('end', function () {
+process.stdin.on('end', function() {
   console.log('parsing commit log:\n', data, '\n');
   var firstLine = data.split('\n').filter(l => l && !l.match(/^\s*Merge/))[0];
   console.log('evaluating commit message:\t', firstLine);
   if (!firstLine.match(/^\w{7} (feat|fix|docs|chore|ci|style|refactor|test|perf)(\([^)]+\))?: .*/gi)) {
-    error('commit message didn\'t match expectation, see cz-conventional-changelog');
+    error("commit message didn't match expectation, see cz-conventional-changelog");
   }
-})
+});
```

Comment:
I think we can get rid of this `bin/validate-commit-msg.js` file as I don't believe its used anymore.

Reply:
It'd like to keep it for now since it seems this was used in conjunction with semantic release which we may re-activate in some time.